openprescribing/dmd2/models.py

```diff
@@ -46,6 +46,12 @@ def __str__(self):
     def title(self):
         return self.nm
 
+    def status(self):
```

Comment:
Can we have a docstring here (and all the other `status` implementations here) explaining that this should return a comma-separated list of ways this item could be filtered out?

Feels like there's some way of refactoring this nicely but I can't think what that is.  One status method called by all the models with switches in it? (yuck) Some way of defining them on the queryset managers?

Reply:
I can't think of a nice way of refactoring this, so I'll leave it as is.